app/controllers/staff/funds_controller.rb

```diff
@@ -18,18 +18,20 @@ def show
 
   def new
     @fund = policy_scope(Fund).new
+    @organisation = Organisation.find(organisation_id)
```

Comment:
Out of curiosity (doesn't necessarily affect mergability!) do we need to add the policy check on to the `Organisation#find` instances here, or is it safe(ish) to assume by this point we know the user can access this organisation? They might have manipulated the URL to get a different Organisation ID into the form/params maybe?

Reply:
Wowza, yes if I change the URL I can hop between organisations unchecked. @leeky should we be authorizing every resource in the scope at each step or perhaps a before filter to authorise any parent resource?

```
Funds
before_filter authorize_org

def create
  …
  authorize @fund
  …
end
```

Reply:
Well, `organisation_id` is user input, so we should probably apply the policy. If you got it through an association, then I'd say not.

Reply:
NB: I wouldn't call `authorize` on the other model, as `Fund` is the main thing we're dealing with here.

Reply:
I now think I understand why I could URL this.. our policy does take care of this concern but  as an `administrator` I don't get halted! https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/policies/fund_policy.rb#L27